saleor/graphql/page/types.py

```diff
@@ -97,3 +98,8 @@ def resolve_available_attributes(root: models.PageType, info, **kwargs):
         return product_models.Attribute.objects.get_unassigned_page_type_attributes(
             root.pk
         )
+
+    @staticmethod
+    @permission_required(PagePermissions.MANAGE_PAGES)
+    def resolve_has_pages(root: models.PageType, info, **kwargs):
+        return root.pages.exists()
```

Comment:
This is fine for now but in the end, it will perform a query for each page type when asking for multiple page types. Is there a way to use dataloader here?

Reply:
I added `TODO` sign here. I'll try to add the dataloader later.